src/Entity/HearingPostRequest.php

```diff
@@ -38,6 +38,16 @@ class HearingPostRequest extends HearingPost
      */
     private $hearingRecipients;
 
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $briefExtraParties;
```

Comment:
Why is this a string value and not a boolean value? It's used as a boolean value.

Reply:
Great point. Updated to be a boolean value called `shouldSendBriefing`.